Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/EditorVertexSelectionBus.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace AzToolsFramework
+{
+    //! Bus to request operations to the EditorVertexSelection classes.
+    class EditorVertexSelectionRequests : public AZ::EBusTraits
+    {
+    public:
+        virtual void DuplicateSelectedVertices() {};
+        virtual void DeleteSelectedVertices() {};
+        virtual void ClearVertexSelection() = 0;
+
+    protected:
+        ~EditorVertexSelectionRequests() = default;
+    };
+    using EditorVertexSelectionRequestBus = AZ::EBus<EditorVertexSelectionRequests>;
+
+    //! Bus to be notified about changes in Editor Vertex Selection state.
+    class EditorVertexSelectionNotifications : public AZ::EBusTraits
```

Comment:
question: I'm not 100% sure this is needed, it would also be useful to pass the new vertex index here which might be useful if we do need it... I'm just wondering why this is needed now and not before?

Reply:
Oh this is a stub, I wanted to disable the vertex actions if no vertex was selected... but I noticed it's not a current feature so I stopped early. I will remove it from this PR.